packages/flutter/lib/src/widgets/sliver_prototype_item_list.dart

```diff
@@ -0,0 +1,188 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
+
+import 'basic.dart';
+import 'framework.dart';
+import 'sliver.dart';
+
+export 'package:flutter/rendering.dart' show RenderSliverFixedExtentBoxAdaptor;
+
+class SliverPrototypeExtentListElement extends SliverMultiBoxAdaptorElement {
```

Comment:
If you keep everything in this file, you should probably make everything except the widget private.